src/views/activity/activity_item_view.js

```diff
@@ -6,7 +6,7 @@ export default BaseView.extend({
   template: ActivityItemView,
 
   events: {
-    'click': 'openUrl',
+    'mousedown': 'openUrl',
```

Comment:
Assuming we land this beautiful PR, do you want to add a `// NOTE:` or `// HACK:` comment here (and in file below) pointing to this bug explaining why we're using `mousedown` instead of `click` event?